src/scr_param.c

```diff
@@ -547,3 +558,16 @@ kvtree* scr_param_set_hash(char* name, kvtree* hash_value)
 
   return kvtree_set(scr_app_hash, name, hash_value);
 }
+
+/* unsets a parameter, returning 0 if the parameter did not exist */
+int scr_param_unset(char* name)
```

Comment:
Can we upgrade this to a ``const char* name``?

Reply:
Done in 28dc2e7 where also `scr_param_set` and similar functions were modified to mark their `name` argument as const (no warnings were produced so they seem to pass them along to const functions on kvtree only).